### ISSUE TITLE
Raspberry Pi - Not load save

### ISSUE THREAD

**Issue by sergiopoverony** [rofl0r/openbor#1]
In retropie installed OpenBor not load saves. In Saves folder i can see saved files. In any games and anytimes not load saves
Thank you

**Comment by rofl0r** [comment]
does retropie use this openbor version ? i doubt it.

**Comment by madmodder123** [comment]
> 
> 
> does retropie use this openbor version ? i doubt it.

Yeah it does: https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/ports/openbor.sh

**Comment by rofl0r** [comment]
probably a path issue. i'd recommend to add (or ask retropie developers to do so) some debug printfs to load/save code so we can debug where it's trying to access the files.
